src/styles/utils/normalize.css

```diff
@@ -0,0 +1,7 @@
+* {
```

Comment:
Browser's default margins are not reset. Add 'margin: 0; padding: 0;' inside the '*' selector to reset them 🛠️